Sources/App/Views/PackageController/GetRoute.Model+ext.swift

```diff
@@ -404,7 +404,7 @@ extension API.PackageController.GetRoute.Model {
             .class("custom-collections"),
             .forEach(customCollections, { collection in
                     .a(
-                        .href(collection.url),  // FIXME: link to custom collection page
+                        .href(SiteURL.collections(.value(collection.name)).relativeURL()),
```

Comment:
Using the name to build the URL works fine but generates URLs like

http://localhost:8080/collections/SSWG%20Graduated

We could make the slug configurable in the `custom-package-collections.json` by adding another field but I've kept it simple for now:

```json
    {
        "name": "SSWG Graduated",
        "description": "SSWG packages that are in 'graduated' state",
        "badge": "SSWG",
        "url": "https://raw.githubusercontent.com/finestructure/sswg-package-lists/refs/heads/main/graduated.json",
        "slug": "sswg-graduated"
    },
```